webrtc.html

```diff
@@ -9671,6 +9691,59 @@ <h2>Dictionary <a class="idlType">RTCStats</a> Members</h2>
       from RTCStats that they indicate, are documented in
       [[!WEBRTC-STATS]].</p>
     </section>
+    <section>
+      <h3><dfn>RTCStatsLifetimeEvent</dfn></h3>
+      <p>The <code><a>statslifetimeended</a></code> event uses
+        the <code><a>RTCStatsLifetimeEvent</a></code>.</p>
+      <div>
+        <pre class="idl">
+          [ Constructor (DOMString type, RTCStatsLifetimeEventInit
+          eventInitDict), Exposed=Window]
+          interface RTCStatsLifetimeEvent : Event {
+            readonly attribute any stat; // Must be RTCStats subtype
+          };</pre>
+        <section>
+          <h2>Constructors</h2>
+          <dl data-link-for="RTCStatsLifetimeEvent"
```

Comment:
Shouldn't it be `RTCStatsLifetimeEndedEvent` ?

Reply:
RTCStatsReport for the win. Then the need for "any" disappears into what we've already done.
Have to make clear in the text that it's *only* the newly-dead data that appears here.

Reply:
Yeah :-( long names!